darknet_ros/test/ObjectDetection.cpp

```diff
@@ -69,14 +69,15 @@ bool sendImageToYolo(ros::NodeHandle nh, const std::string& pathToTestImage) {
   }
 
   // Get test image
-  cv_bridge::CvImagePtr cv_ptr(new cv_bridge::CvImage);
-  cv_ptr->image = cv::imread(pathToTestImage, CV_LOAD_IMAGE_COLOR);
-  cv_ptr->encoding = sensor_msgs::image_encodings::RGB8;
-  sensor_msgs::ImagePtr image = cv_ptr->toImageMsg();
+  auto header = std_msgs::Header();
+  header.stamp = ros::Time::now();
+  header.seq = seq;
```

Comment:
without a distinct sequence number, the logic for preventing duplicate image detections fails